ContactList/ContactList/TableViewCell.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  TableViewCell.swift
+//  ContactList
+//
+//  Created by Maksym Humeniuk on 5/7/19.
+//  Copyright © 2019 Maksym Humeniuk. All rights reserved.
+//
+
+import UIKit
+
+class TableViewCell: UITableViewCell {
+
+    @IBOutlet weak var cellImage: UIImageView!
+    
+    @IBOutlet weak var cellNameLabel: UILabel!
+    
+    @IBOutlet weak var cellContactLabel: UILabel!
+    
+    func updateWith (contact: Person) {
+        cellImage.image = contact.image ?? Constants.emptyAvatar
```

Comment:
Leave as it is for now.
In Iteration 2 please move this logics into view model to avoid any calculations in view/cell